notebooks/noj_book/automl.clj

```diff
@@ -112,11 +115,12 @@ ctx-after-train
 ;;
 (vals ctx-after-train)
 
-;; The `model` function has closed over the id, so is knows "his id", so in the `transform`
-;; mode it can get the data created at `:fit`.  So the `model` function can "send" data to itself
-;; from `:fit` to `:transform`, the `trained model`.
+;; The `model` function has closed over the id, so is knows "its id", so in the
```

Comment:
Great explanation.

Just noticed a small typo: 
`so is knows`  -> `so it knows`